src/patternfly/components/Chip/chip.scss

```diff
@@ -1,51 +1,40 @@
 .pf-c-chip {
+  --pf-c-chip--PaddingTop: var(--pf-global--spacer--form-element);
+  --pf-c-chip--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-chip--PaddingBottom: var(--pf-global--spacer--form-element);
   --pf-c-chip--PaddingLeft: var(--pf-global--spacer--sm);
   --pf-c-chip--BackgroundColor: var(--pf-global--Color--light-100);
-  --pf-c-chip--BorderColor: var(--pf-global--secondary-color--100);
-  --pf-c-chip--BorderWidth: var(--pf-global--BorderWidth--sm);
-  --pf-c-chip--BorderRadius: var(--pf-global--BorderRadius--sm);
-
-  // overflow chip
-  --pf-c-chip--m-overflow--BackgroundColor: var(--pf-global--BackgroundColor--200);
-  --pf-c-chip--m-overflow--PaddingLeft: 0;
-  --pf-c-chip--m-overflow--BorderWidth: 0;
+  --pf-c-chip--before--BorderColor: var(--pf-global--BorderColor--300);
+  --pf-c-chip--before--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-chip--before--BorderRadius: var(--pf-global--BorderRadius--sm);
 
   // overflow chip button
-  --pf-c-chip--m-overflow--c-button--BorderRadius: var(--pf-global--BorderRadius--sm);
-  --pf-c-chip--m-overflow--c-button--BorderWidth: 0;
-  --pf-c-chip--m-overflow--c-button--PaddingLeft: var(--pf-global--spacer--sm);
-  --pf-c-chip--m-overflow--c-button--PaddingRight: var(--pf-global--spacer--sm);
-  --pf-c-chip--m-overflow--c-button--hover--BorderWidth: var(--pf-global--BorderWidth--sm);
-  --pf-c-chip--m-overflow--c-button--hover--BorderColor: var(--pf-global--secondary-color--100);
-  --pf-c-chip--m-overflow--c-button--active--BorderWidth: var(--pf-global--BorderWidth--sm);
-  --pf-c-chip--m-overflow--c-button--active--BorderColor: var(--pf-global--secondary-color--100);
-  --pf-c-chip--m-overflow--c-button--focus--BorderWidth: var(--pf-global--BorderWidth--sm);
-  --pf-c-chip--m-overflow--c-button--focus--BorderColor: var(--pf-global--secondary-color--100);
-
-  // read-only chip
-  --pf-c-chip--m-read-only--PaddingTop: var(--pf-global--spacer--form-element);
-  --pf-c-chip--m-read-only--PaddingRight: var(--pf-global--spacer--sm);
-  --pf-c-chip--m-read-only--PaddingBottom: var(--pf-global--spacer--form-element);
+  --pf-c-chip--m-overflow--hover--before--BorderColor: var(--pf-global--secondary-color--100);
```

Comment:
I'm not sure about the use of `secondary-color--100`. That's black-600. Should we use `pf-global--BorderColor--200` instead (black-500) so it's more inline with our use of border colors?

Reply:
good point.

@mceledonia does this look ok to you?

<img width="351" alt="Screen Shot 2020-04-13 at 1 43 35 PM" src="https://user-images.githubusercontent.com/20118816/79144368-d8896d80-7d8c-11ea-8a11-1ae7263101af.png">